src/deploy/osp_deployer/profiles/csp.json

```diff
@@ -249,8 +257,13 @@
                ]
             },
          "use_ipmi_driver":"",
-         "use_in_band_introspection":"",
```

Comment:
It doesn't look to me like the changes in this block are related to this patch.  We should undo these.